pkg/extensions/enterprise_imports.go

```diff
@@ -53,4 +52,5 @@ import (
 	_ "github.com/grafana/e2e"
 	_ "github.com/grafana/gofpdf"
 	_ "github.com/grafana/gomemcache/memcache"
+	_ "github.com/spyzhov/ajson"
```

Comment:
weird, why did this get moved from the 3rd-party section to the grafana section?